features/full_tests/batch_1/in_foreground.feature

```diff
@@ -7,7 +7,8 @@ Scenario: Test handled exception after delay
     And the event "app.inForeground" is true
 
 # Skip due to an issue on later Android platforms - [PLAT-5464]
-@Flaky @skip_android_11 @skip_android_10
+# Possible @Flaky
```

Comment:
Is this scenario flaky?

Reply:
Possibly.  😁  I think I left that in by mistake - now removed.